openqaoa/workflows/optimizer.py

```diff
@@ -672,6 +680,14 @@ def _serializable_dict(self, complex_to_string:bool = False):
         # we add the keys of the QAOA object that we want to return
         serializable_dict['data']['input_parameters']['circuit_properties'] = dict(self.circuit_properties)
 
+        # include parameters in the header metadata
+        serializable_dict['header']['metadata']['circuit_param_type'] = serializable_dict['data']['input_parameters']['circuit_properties']['param_type']
```

Comment:
Shall we put these into a function? So that the optimiser file doesn't blow up in size?